### ISSUE TITLE
Character encoding error in self.address

### ISSUE THREAD

**Issue by jo45** [costastf/locationsharinglib#17]
Hello.

I think there's a character encoding problem in self.address:

Traceback (most recent call last):
  File "../../locationsharinglib/test.py", line 6, in <module>
    print(person)
  File "/opt/locationsharinglib/locationsharinglib/locationsharinglib.py", line 108, in __str__
    'Current location :{}'.format(self.address),
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 28: ordinal not in range(128)

Not sure if related but adresses concerned are located in France.

**Comment by costastf** [comment]
Do you mind editing that line on your local file and adding a u before the quotes? so instead of 
```'Current location :{}'.format(self.address)```

make it like 
```u'Current location :{}'.format(self.address)```.
If that fixes your issue I will make all text into unicode and bump the version.

**Comment by costastf** [comment]
And if you could add ```from __future__ import unicode_literals``` in line 34 and test that too, which would make all string litterals to unicode and report back, that would be great!

**Comment by jo45** [comment]
I tried both of your suggestions (one at a time and both at the same time), I still get the error.

Solved the issue by adding
```
import sys
reload(sys)
sys.setdefaultencoding('utf8')
```
With this added, I don't have any errors anymore, and it prints the City name correctly with "é" characters in "Current location" (that was what caused the error it seems).
Not sure it's the way to go as I'm not a developper at all.

Anyway, I still get place name or coordinates randomly as "Current location". From what I understand, it is provided by Google so we can't do anything about that.

I spoted another thing: I also get my own location data, but "Fullname" and "Nickname" is replaced by Google login (I also think it's Google Maps normal behavior).
